daemon/snap.go

```diff
@@ -240,6 +240,8 @@ func mapLocal(about aboutSnap, sd clientutil.StatusDecorator) *client.Snap {
 		result.GatingHold = &about.gatingHold
 	}
 
+	result.InhibitedRefresh = snapst.RefreshInhibitedTime != nil
```

Comment:
`daemon/export_api_snaps_test.go` expose `mapLocal` for testing which is used in `daemon/api_snaps_test.go`
It looks like `TestMapLocalFields` requires an update for the new field.